doc/project/checklist.md

```diff
@@ -120,19 +120,12 @@ The IP block is synthesized as part of the continuous integration checking and m
 
 All CDC synchronization flops use behavioral synchronization macros (e.g. `prim_flop_2sync`) not manually created flops.
 
-### SEC_CM_IMPLEMENTED
-
-Any appropriate security counter-measures documented and implemented.
-For redundantly encoded FSMs, [the sparse-fsm-encode.py script](https://github.com/lowRISC/opentitan/blob/master/util/design/sparse-fsm-encode.py) must be used to generate the encoding.
-
-### SEC_NON_RESET_FLOPS
-
-A review of sensitive security-critical storage flops was completed.
-Where appropriate, non-reset flops are used to store secure material.
-
-### SEC_SHADOW_REGS
+### SEC_CM_DOCUMENTED
 
-Shadow registers are implemented for all appropriate storage of critical control functions.
+Any appropriate security counter-measures documented and reviewed.
+The implementation of security countermeasures can be delayed until D3 if:
+- The addition of those countermeasures would not have a material impact on area, and
```

Comment:
Should we be more specific about the extent of the "negative impact"? E.g.:
```
- Any negative impact of those countermeasures on PPA metrics would remain below 10%,
```
Also, since many countermeasures have noticeable impact on PPA, should we allow insertion of dummy logic at this point to account for PPA impact so that we have a way to declare D2 even with countermeasures that would have an impact larger than 10%?

Reply:
I'll add that wording